src/main/java/com/truelayer/java/TrueLayerClient.java

```diff
@@ -23,22 +23,24 @@
  */
 @AllArgsConstructor
 public class TrueLayerClient implements ITrueLayerClient {
+
     private IAuthenticationHandler authenticationHandler;
-    private IPaymentsApi paymentsHandler;
+    private IPaymentsHandler paymentsHandler;
     private IPaymentsProvidersHandler paymentsProvidersHandler;
     private IMerchantAccountsHandler merchantAccountsHandler;
     private IMandatesHandler mandatesHandler;
-    private IPayoutsApi payoutsHandler;
+    private IPayoutsHandler payoutsHandler;
+    private ICommonHandler commonHandler;
+
     private IHostedPaymentPageLinkBuilder hostedPaymentPageLinkBuilder;
-    private ICommonApi commonApi;
 
     public TrueLayerClient(
             IAuthenticationHandler authenticationHandler,
             IHostedPaymentPageLinkBuilder hostedPaymentPageLinkBuilder,
-            ICommonApi commonApi) {
+            ICommonHandler commonApi) {
```

Comment:
Nit: Looks like this parameter's name wasn't modified to match its new type.

Reply:
Thank you !